tf2onnx/tfonnx.py

```diff
@@ -2486,19 +2508,7 @@ def process_tf_graph(tf_graph, continue_on_error=False, verbose=False, target=No
     if custom_rewriter is not None:
         rewriters.extend(custom_rewriter)
 
-    try:
-        ops = g.get_nodes()
-        for rewrite in rewriters:
-            ops = rewrite(g, ops)
-            g.set_nodes(ops)
-    except Exception as ex:
-        type_, value_, traceback_ = sys.exc_info()
-        log.error("node %s: exception %s" % (rewrite, ex))
-        ex_ext = traceback.format_exception(type_, value_, traceback_)
-        if continue_on_error:
-            log.info(ex_ext)
-        else:
-            raise ex
+    tensorflow_onnx_rewrite(g, rewriters)
```

Comment:
there is a run_late_rewriters, which are doing similar things, can we re-use? 
@pengwa for more comments.

Reply:
so the question is does the graph processing order matter?
rewrite parent then child vs. rewrite child then parent?